utility_analysis_new/dp_engine.py

```diff
@@ -55,12 +55,12 @@ def _create_compound_combiner(
         budget = self._budget_accountant.request_budget(
             mechanism_type, weight=aggregate_params.budget_weight)
         compound_combiners = []
-        if aggregate_params.metrics in [pipeline_dp.Metrics.COUNT]:
+        if [pipeline_dp.Metrics.COUNT] in aggregate_params.metrics:
```

Comment:
It should be 
`if pipeline_dp.Metrics.COUNT in aggregate_params.metrics:`

if we use [] around COUNT, the expression can't be true in our case:
operator in it's operator of element inclusion, e.g.

 `2 in [1,2,3]` is true, but 

`[2] in [1,2,3]` is false, because [2] is not in [1,2,3], while 

`[2] in [1, [2], 3])` is true

Reply:
Got it, thank you for the explanation!